.github/workflows/cherry-pick.yml

```diff
@@ -1,4 +1,4 @@
-name: Cherry Pick
+name: Cherry pick
```

Comment:
Hmm, I've never used this, and I currently manage all of our v1 releases...maybe it's time to get rid of it?

Reply:
Removed. There's also the `dependencies-check.yml` workflow, which is run every 4 days but [has been failing](https://github.com/pydantic/pydantic/actions/workflows/dependencies-check.yml) for quite some time now. I think it should be replaced by a new workflow, running tests with the [uv resolution strategy](https://docs.astral.sh/uv/concepts/resolution/#resolution-strategy) set to `lowest`.